src/proc.edu.ria.es6.js

```diff
@@ -1,36 +1,28 @@
-import 'babel-polyfill';
-import cfg from './config.es6.js';
-import THREE from 'three';
-import Camera from './camera.es6.js';
-import { scene, renderer, render } from './init.es6.js';
-import { HemisphereLight, HemisphereLightHelper, DirectionalLightHelper } from './light.es6.js';
-import scan from './scan.es6.js';
-import generator from './generator.es6.js';
-import * as renderMethods from './methods.es6.js';
-import mergeGeometry from './mergeGeometry.es6.js';
-import pilar from './pilar.es6.js';
+import "babel-polyfill";
+import cfg from "./config.es6.js";
+import THREE from "three";
+import Camera from "./camera.es6.js";
+import { scene, renderer } from "./init.es6.js";
+import sceneAdd from "./sceneAdd.es6.js";
+import getCityPilar from "./getCityPilar.es6.js";
+import generator from "./generator.es6.js";
+import putBlock from "./putBlock.es6.js";
+import getCityBlockHelper from "./getCityBlockHelper.es6.js";
+import mergeGeometry from "./mergeGeometry.es6.js";
 
+var renderMethods = { putBlock };
 var camera = new Camera();
 
-let world = {
-  seed: cfg.seed,
-  seedHash: cfg.seedHash,
-  size: cfg.size || 4,
-  unit: cfg.unit || 1,
-  cubicSize: cfg.cubicSize,
-  scan: scan(cfg.size, 1),
-  data: []
-};
-
-// let geometry = new THREE.BufferGeometry();
-// geometry.dynamic = true;
-
 var update = () => {
   cfg.delta = cfg.clock.getDelta();
 
   if (cfg.debug) {
-    HemisphereLightHelper.update();
-    DirectionalLightHelper.update();
+    if (HemisphereLightHelper) {
+      HemisphereLightHelper.update();
+    }
+    if (DirectionalLightHelper) {
+      DirectionalLightHelper.update();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["DirectionalLightHelper" is not defined.](https://www.codacy.com/app/lagora-franck/proc-edu-ria/file/2546663684/issues/source?bid=3286583&fileBranchId=3287652#l24)